tckprocess.adoc

```diff
@@ -0,0 +1,422 @@
+via the specification's issue tracker using the label
+`certification` and include the following information:
+
+-   Statement of Acceptance of the terms of the EFTL
+
+-   Product Name, Version and download URL (if applicable)
+
+-   Specification Name, Version and download URL
+
+-   TCK Version, digital SHA-256 fingerprint and download URL
+
+-   Implementation runtime Version(s) tested
+
+-   Public URL of TCK Results Summary
+
+-   Any Additional Specification Certification Requirements
+
+-   Java runtime used to run the implementation
+
+-   Summary of the information for the certification environment,
+    operating system, cloud, ...​
+
+-   A statement attesting that all TCK requirements have been met,
+    including any compatibility rules
+
+[#_additional_specification_certification_requirements]
+### Additional Specification Certification Requirements
+
+Specification's may require additional items for a Certification
+Request as defined in their corresponding TCK Documentation under the
+section labeled \"Additional Certification Requirements\".
+
+Examples of such additional requirements may include:
+
+-   Name and version of Compatible Implementation used for
+    interoperability tests
+
+-   Name and version of Databases used for persistence tests
+
+-   Name and version of NoSQL implementations used in persistence tests
+
+[#_public_tck_results_summary]
+### Public TCK Results Summary
+
+While certification is on your honor, the community MUST be able to see
+your test results summary. At a minimum a results summary MUST:
+
+-   Be publicly visible with no password protection or sign-up
+
+-   Include a Summary Page containing:
+
+    -   All information in the above Certification Request
+
+    -   The Total number of tests run and passed. If you are running a
+        platform TCK, this means reporting the tests run for each
+        required TCK, which includes the platform TCK and some of the
+        individual TCKs that are not integrated with the platform TCK.
+
+An optional "Test List Page" showing all tests run may be linked from
+the Summary Page. The Summary Page URL is the URL that MUST be included
+in any Certification Requests.
+
+The following are explicitly not requirements:
+
+-   The Ability for the public to run the tests themselves
+
+-   Full log output of the TCK
+
+Implementors may supply this information and provide support for how to
+run a TCK against their implementation, but it is not required.
+
+[#_certification_resolution]
+### Certification Resolution
+
+As noted, certification requests do not in themselves grant rights to
+use the MicroProfile Platform Compatible logo. Approval that the TCK
+requirements have been met is a prerequisite for obtaining
+MicroProfile Platform Compatible logo usage for profile
+certification. The required approval processes is:
+
+-   Approval by lazy consensus after a period of two weeks (14 days)
```

Comment:
Why 14 days? Should we do with 7 days for normal votes as this is not about releasing specs?

Reply:
Maybe 2 committers not from the same company as the one who submitted the request. 72 hours window.

Reply:
compatibility request is to made on microprofile-wg.